src/XcooBee/Http/Response.php

```diff
@@ -55,8 +60,64 @@ public static function setFromHttpResponse(ResponseInterface $response)
         if($xcoobeeResponse->code === 200 && $xcoobeeResponse->errors){
             $xcoobeeResponse->code = 400;
         }
-
+        
         return $xcoobeeResponse;
     }
-
+    
+    /**
+     * 
+     * name: has next page
+     * 
+     * @param ResponseInterface $resultObject
+     * 
+     * @return \XcooBee\Http\Response
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function hasNextPage($resultObject)
+    {
+        foreach($resultObject->result as $result){
+            self::$endCursor = $result->page_info->end_cursor;
+            return $result->page_info->has_next_page;
+        }
+    }
+    
+    /**
+     * 
+     * name: get next page data
+     * 
+     * @param ResponseInterface $resultObject
+     * 
+     * @return \XcooBee\Http\Response
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getNextPage($resultObject)
+    {
+        self::$response = $resultObject;
+        $request = new Request();
```

Comment:
`Request` with all needed headers and variables should be set to property of response